packages/dropdowns/src/index.ts

```diff
@@ -28,5 +28,5 @@ export { MediaItem } from './elements/Menu/Items/MediaItem';
 export { NextItem } from './elements/Menu/Items/NextItem';
 export { PreviousItem } from './elements/Menu/Items/PreviousItem';
 export type { GARDEN_PLACEMENT, POPPER_PLACEMENT } from './utils/garden-placements';
-
+export type { ILabelProps } from './elements/Fields/Label';
```

Comment:
Do we need to export all interfaces from these subcomponents in Dropdown retroactively? 🤔

Reply:
All applicable element interfaces should be exported. See https://github.com/zendeskgarden/react-components/blob/main/packages/buttons/src/index.ts